app/views/people/_form.html.erb

```diff
@@ -48,6 +48,9 @@
     <%= f.input :start_date, :as => :string, :input_html => { :class => 'datepicker', :size => 11 }  %>
     <%= f.input :division1, :as => :select, :label => 'Division', :collection => Person::DIVISION1 %>
     <%= f.input :division2, :as => :select, :label => 'Squad', :collection => Person::DIVISION2 %>
+    <h4> Personnel Image </h4>
+    <%= @person.portrait %>
```

Comment:
As it is now, when a person has a portrait, it shows the portrait's filename, but if someone already has a portrait, it might be better to show the picture itself), with the option to change the picture, and possibly to delete it.